src/validators/literal.rs

```diff
@@ -75,7 +75,8 @@ impl Validator for LiteralSingleStringValidator {
         _slots: &'data [CombinedValidator],
         _recursion_guard: &'s mut RecursionGuard,
     ) -> ValResult<'data, PyObject> {
-        let either_str = input.strict_str()?;
+        let strict = if let Some(s) = _extra.strict { s } else { false };
```

Comment:
```suggestion
        let strict = extra.strict.unwrap_or(false);
```
and change the parameter name to `extra`.

Reply:
I've fixed this.